docs/guides/managing_packages.md

```diff
@@ -0,0 +1,122 @@
+
+Now that we have all we need, publish the new agent and all its dependencies to the local registry:
+
+```bash
+autonomy publish --local --push-missing
+```
+
+The agent and its dependencies live now in the local registry, so we can delete the temporary agent:
+
+```bash
+cd ..
+autonomy delete my_agent
+```
+
+We will focus now on developing inside the local registry. First of all, it is useful to add other vendor paths to the `.gitignore` (for example `packages/valory` and `packages/open-aea`) so only your packages are tracked. Secondly, even if the skill is now in the local registry, there is another task we need to perform: update the `packages/packages.json` file. This file contains a list of all packages in the registry, as well as their hashes. It's useful to detect when packages change, are added or removed. If you open that file you will see it is empty. To reflect our latest changes, run:
+
+```bash
+autonomy packages lock
+```
+
+This will add the new skill hashes to it:
+
+```json
+{
+    "protocol/valory/abci/0.1.0": "bafybeiaw3tzlg3rkvnn5fcufblktmfwngmxugn4yo7pyjp76zz6aqtqcay",
+    "connection/valory/abci/0.1.0": "bafybeih34yj6l7akh6wfhgwaud3luvmwxkiakcqzv3zjiz5q7jfqtdnela",
+    "protocol/valory/contract_api/1.0.0": "bafybeidyjrthhq3p27k3v3a4xx7b7lu4fe4h765gmkyyyj7xur4b25bxim",
+    "protocol/valory/http/1.0.0": "bafybeibfjhmmjut4dukhunbcmp5zi4eqsk6rgreztmg4svqfzyq4ei3o7i",
+    "protocol/valory/ledger_api/1.0.0": "bafybeifawyt7ayxrvpagqecowx6y5hvc3dxzjbhhkutxsuk7u54cxo2g54",
+    "connection/valory/http_client/0.1.0": "bafybeiet6defg3qvov4r2ucjtkywkz53xcg6mgarjfdqmnxfgdixgcx45y",
+    "connection/valory/ledger/0.1.0": "bafybeicup3pqzi74h67z6oxzw7qhsz2dpezeqhalg7bex37fqcx5koivqu",
+    "contract/valory/service_registry/0.1.0": "bafybeicczwyiwjiejqtt2kvkx3eit6faidt3zv5tkfdjyczg3e5xyhtvsy",
+    "protocol/open_aea/signing/1.0.0": "bafybeiambqptflge33eemdhis2whik67hjplfnqwieoa6wblzlaf7vuo44",
+    "protocol/valory/tendermint/0.1.0": "bafybeihcnjhovvyyfbkuw5sjyfx2lfd4soeocfqzxz54g67333m6nk5gxq",
+    "skill/valory/abstract_abci/0.1.0": "bafybeidt4cyfhlla6faca6us6law7dijrlrzihcj6uwrmeaa3ho6fdck2u",
+    "skill/valory/abstract_round_abci/0.1.0": "bafybeigpz64qx22a4zzlcczmc5frhlecytrh5q3x77lwe3a5kee2qrcchm",
+    "skill/john_doe/my_skill/0.1.0": "bafybeiaecrvczomydblyhpxsflczkkwlwgu4wszydcwbhco3jjxdlcurve",
+    "agent/john_doe/my_agent/0.1.0": "bafybeicauz6f4x7nvpcxze3peusptd5jeinuum3mhtije23jyvbizdw42y"
+}
+```
+
+While developing, each change will make this hashes out of date. You can run the following to check whether you need to update `autonomy packages lock` again:
+
+```bash
+autonomy packages lock --check
+```
+
+Each time you need a new package you will need to either copy/paste/edit an existing one and edit the agent's configuration file or use the intermediate agent method discussed before.
```

Comment:
You can just add the hash to the packages json and pull that way too, using `autonomy packages sync`